test/link/table.txt

```diff
@@ -44,11 +44,11 @@ Elem:
 Custom:
  - name: "reloc.Elem"
   - section: Elem
-   - R_FUNC_INDEX_LEB   idx=0    addend=0    offset=0x6(file=0x38)
```

Comment:
Hm, I guess I'm just concerned that this will break otherwise. But if there's no easy way to do it, I guess it's OK. Do you mind creating an issue for it though?